src/ParallelAlgorithms/Events/ObserveFields.hpp

```diff
@@ -142,11 +161,13 @@ class ObserveFields<VolumeDim, ObservationValueTag, tmpl::list<Tensors...>,
 
   explicit ObserveFields(const std::string& subfile_name,
                          const std::vector<std::string>& variables_to_observe =
-                             {db::tag_name<Tensors>()...},
+                             VariablesToObserve::default_value(),
```

Comment:
This shouldn't compile.. does it? It should be `suggested_value`

Reply:
I think with a rebase now this default should disappear complete. I might have missed it but we should delete it here

Reply:
I don't understand why this compiles. Is the default value never used and thus the compiler never tries to evaluate this piece of code? In any case, please delete the default.

Reply:
I don't understand why this compiles. Is the default value never used and thus the compiler never tries to evaluate this piece of code? In any case, please delete the default.